modules/ROOT/pages/create-instance-task.adoc

```diff
@@ -3,50 +3,58 @@ ifndef::env-site,env-github[]
-+
-NOTE: OpenAPI Specification (OAS) 3.0 is currently not supported in API Manager. You must convert the API to RAML or OAS 2.0 to manage it using API Manager. For converting the specifications to OAS 2.0, you must use third-party tools, such as APImatic.
+. From *API name*, enter `jsonplaceholder`.
 +
-. From *API Version*, select version (`1.0`).
-. From *Asset Version*, select the Exchange asset version.
-. From *Managing Type*, choose *Basic Endpoint* or *Endpoint with Proxy*. 
+NOTE: OpenAPI Specification (OAS) 3.0 is supported, with the exception of the callback feature. To work around this issue, handle the callback outside of the Mule runtime engine domain or use an OAS 3.0 specification that does not use callbacks.
 +
-For this example, select *Endpoint With Proxy*.
-. In *Implementation URI*, enter the URL that hosts your web site or an arbitrary URL, such as the example shown (`+http://api.domain.com+`). 
+. From *API version*, select version (`1.0`).
+. From *Asset version*, select the Exchange asset version.
+. From *Managing type*, select *Endpoint with proxy*.
+. In *Implementation URI*, enter your web site URL or an arbitrary URL, such as `+http://api.domain.com+`. 
 +
 If you choose _Endpoint with Proxy_ and do not make an entry (such as accepting the default value), you cannot save your configuration.
-. From *Client provider*, select the IdP that you want to use to authenticate access to this API from registered client applications:
+. From *Client provider*, select the client provider to authenticate API access from registered client applications:
 +
 .Associate client provider with the API
 image::create-instance-endpoint-midp.png[65%,65%, Create instances with multiple client provider]
 +
-<1> The *Tag* field.
-<2> The *Managing Type* field. Select *Endpoint With Proxy*.
-<3> The *Client provider* field is at the bottom of the API Configuration section.
+* The first item under API Configuration is the *Add a Tag* field.
+* The second item shows the *Managing Type* field. Select *Endpoint With Proxy*.
+* The third item shows the *Client provider* field. 
 +
-If no client providers are set up for your environment and organization, this field is not visible.
+If no client providers are configured for your environment and organization, the *Client provider* field is not visible.
+
 . Select a proxy deployment target, for example, CloudHub.
 . Accept the default path `/` for the proxy endpoint or enter another path.
-. Save, or optionally set advanced options, and then save.
-. In Advanced Options, optionally assign an instance label for the proxy, or accept the default label, and set a response timeout.
+. In *Advanced options*, optionally assign an instance label for the proxy (or accept the default label), and set a response timeout.
+. Click *Save*.
 +
-. Return to the API administration dashboard: Click *API Administration*.
+. Click *API Administration* to return to the API Administration dashboard.
```

Comment:
If API Administration is a label, boldface it.

Reply:
Fixed.